README.md

```diff
@@ -31,82 +69,134 @@ Add this plugin like this:
 
 ## Example usage
 
-Let's start by creating a **root meta feed**, necessary for using this module.
-It lives alongside your existing "classical" feed, which we'll refer to as
-**main feed**.
-
+We create a subfeed for messages which describe us under our `root` feed
 ```js
-sbot.metafeeds.findOrCreate((err, metafeed) => {
-  console.log(metafeed) // { seed, keys }
+const find = (feed) => (
+  feed.feedpurpose === 'aboutMe' &&
+  feed.feedformat === 'classic'
+)
+const details = {
+  feedpurpose: 'aboutMe',
+  feedformat: 'classic',
+}
+
+sbot.metafeeds.findOrCreate(null, find, details, (err, aboutMeFeed) => {
+  console.log(rootFeed)
+
+  // publish things to aboutMeFeed!
 })
 ```
 
-Now this has created the `seed`, which in turn is used to generate an [ssb-keys]
-`keys` object. The `seed` is actually also published on the _main_ feed as a
-private message to yourself, to allow recovering it in the future.
+Once you have these the *FeedDetail* object `aboutMeFeed` you can publish to that feed:
 
-There can only be one _root_ meta feed, so even if you call `findOrCreate` many
-times, it will not create duplicates, it will just load the meta feed
-`{ seed, keys }`.
+```js
+const content = {
+  type: 'about',
+  about: rootFeed.keys.id,
+  name: 'baba yaga'
+  description: 'lives in a hutt in the forest, swing by sometime!'
+}
+sbot.db.publishAs(aboutMeFeed.keys, content, cb)
+```
+
+
+## API
+
+### `sbot.metafeeds.findOrCreate(cb)`
```

Comment:
wrote and explicit signature documentation for this special usage to make it clear.
I think this should be moved down in the docs

:fire:
I disliked that the object you got back from this call of this method was different, so I made it isomorphic:
- metafeed: null
- feedpuprpose: 'root'
- feedtype: 'bendybutt-v1',
- metadata: {}

NOTE I think we should change `feedtype` to `format`

Reply:
addressed in https://github.com/ssbc/ssb-meta-feeds/pull/62